pkg/controller/che/che_controller_test.go

```diff
@@ -1664,6 +1665,150 @@ func TestShouldSetUpCorrectlyInternalCheServerURL(t *testing.T) {
 	}
 }
 
+func TestShouldSetUpCorrectlyWebsocketEndpoint(t *testing.T) {
```

Comment:
Pls. Have a look at 
https://github.com/eclipse/che-operator/blob/master/pkg/deploy/server/che_configmap_test.go#L103
We don't use any reconciliation there to check ConfigMap data. 
Pls update the test and move to that file